tests/page/expect-misc.spec.ts

```diff
@@ -220,37 +220,45 @@ test.describe('toHaveClass', () => {
     const error = await expect(locator).toHaveClass(['foo', 'bar', /[a-z]az/], { timeout: 1000 }).catch(e => e);
     expect(error.message).toContain('expect.toHaveClass with timeout 1000ms');
   });
+});
 
-  test('allow matching partial class names', async ({ page }) => {
-    await page.setContent('<div class="foo bar"></div>');
+test.describe('toContainClass', () => {
+  test('pass', async ({ page }) => {
+    await page.setContent('<div class="foo bar baz"></div>');
     const locator = page.locator('div');
-    await expect(locator).toHaveClass('foo', { partial: true });
-    await expect(locator).toHaveClass('bar', { partial: true });
-    await expect(
-        expect(locator).toHaveClass(/f.o/, { partial: true })
-    ).rejects.toThrow('Partial matching does not support regular expressions. Please provide a string value.');
-    await expect(locator).not.toHaveClass('foo');
-    await expect(locator).not.toHaveClass('foo', { partial: false });
-    await expect(locator).toHaveClass('  bar   foo ', { partial: true });
-    await expect(locator).not.toHaveClass('does-not-exist', { partial: true });
-    await expect(locator).not.toHaveClass('  baz   foo ', { partial: true }); // Strip whitespace and match individual classes
+    await expect(locator).toContainClass('');
+    await expect(locator).toContainClass('bar');
+    await expect(locator).toContainClass('baz bar');
+    await expect(locator).toContainClass('  bar   foo ');
+    await expect(locator).not.toContainClass('  baz   not-matching '); // Strip whitespace and match individual classes
+  });
 
-    await page.setContent('<div class="foo bar baz"></div>');
-    await expect(locator).toHaveClass('foo bar', { partial: true });
-    await expect(locator).toHaveClass('', { partial: true });
+  test('pass with SVGs', async ({ page }) => {
```

Comment:
Is this important?

Reply:
SVGs have no `class` attribute and they only have `classList` so yes. Was related to some fix around it.